core/http/routes/ui.go

```diff
@@ -86,6 +87,18 @@ func RegisterUIRoutes(app *fiber.App,
 	}
 
 	app.Get("/", localai.WelcomeEndpoint(appConfig, cl, ml, modelStatus))
+	app.Get("/login", func(c *fiber.Ctx) error {
```

Comment:
shall we reserve GET `/login` to render `views/login`?

Would make also sense so we can move this route from a `GET` instead as a `POST` , or call it `/auth` otherwise.

Can't we otherwise set the cookies from JS? What do you think?

Reply:
yeah, there are some room for simplification 
i'll update pr soon

Reply:
done
cookies now set through js and the backend part mostly untouched
it just support cookies now